cmake/Codegen.cmake

```diff
@@ -43,7 +43,7 @@ configure_file(
 install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/../caffe2
         DESTINATION include
         FILES_MATCHING PATTERN "*.h")
-if (BUILD_ATEN_MOBILE)
+if (BUILD_ATEN_MOBILE AND NOT BUILD_TORCH_MOBILE)
```

Comment:
Can we just remove this completely? Seems like `BUILD_MOBILE` will  now just do the standard thing. No need for ATen core.

Reply:
Probably still need this condition as we keep distinction between caffe2 mobile and libtorch mobile. Probably should combine this one into "NOT INTERN_BUILD_ATEN_OPS" as well? if it's not building aten ops, then only install aten/core; otherwise doesn't matter?